widgetssdk/src/main/java/com/glia/widgets/callvisualizer/ActivityWatcherForCallVisualizer.kt

```diff
@@ -128,326 +94,245 @@ class ActivityWatcherForCallVisualizer(
                 TAG, "Activity does not support ActivityResultRegistry APIs, " +
                         "legacy onActivityResult() will be used to acquire a media projection token"
             )
-            startMediaProjectionLaunchers.remove(activity::class.simpleName)
+            controller.removeMediaProjectionLaunchers(activity::class.java.simpleName)
```

Comment:
this might be confusing, because the method takes String as a key, maybe it would be better to pass activity instead?
also I see different classNames are used in different places `activity::class.java.simpleName`, `activity::class.simpleName`

Reply:
activity::class.java.simpleName and activity::class.simpleName return the same value, so might as well use the shorter one. I prefer keeping the view references in controller to the minimum, thats why the change. This way we are passing on exactly what we need and nothin more